camply/providers/going_to_camp/going_to_camp_provider.py

```diff
@@ -503,41 +526,38 @@ def list_site_availability(
         available_sites: List[AvailableResource]
             The list of available sites
         """
-        search_filter = {
-            "mapId": campground.map_id,
-            "resourceLocationId": campground.facility_id,
-            "bookingCategoryId": 0,
-            "startDate": start_date.isoformat(),
-            "endDate": end_date.isoformat(),
-            "isReserving": True,
-            "getDailyAvailability": False,
-            "partySize": 1,
-            "numEquipment": 1,
-            "equipmentCategoryId": NON_GROUP_EQUIPMENT,
-            "filterData": [],
-        }
+        search_filter = SearchFilter(
+            mapId=campground.map_id,
+            resourceLocationId=campground.facility_id,
+            bookingCategoryId=0,
+            startDate=start_date.isoformat(),
+            endDate=end_date.isoformat(),
+            isReserving=True,
+            getDailyAvailability=False,
+            partySize=1,
+            numEquipment=1,
+            filterData=[],
+        )
         if equipment_type_id:
-            search_filter["subEquipmentCategoryId"] = equipment_type_id
-
+            search_filter.equipmentCategoryId = NON_GROUP_EQUIPMENT
+            search_filter.subEquipmentCategoryId = equipment_type_id
```

Comment:
Here's where we only use the `equipmentCategoryId` when `subEquipmentCategoryId` is also used.

Reply:
Was the bug that "group sites" were ending up in the result set? 

Assuming `equipment_type_id` is a non-group equipment type, the addition of the `NON_GROUP_EQUIPMENT` seems like it should be superfluous here.